runtime/cudaq/cudaq.cpp

```diff
@@ -19,6 +19,7 @@
 #include "distributed/mpi_plugin.h"
 #include <dlfcn.h>
 #include <filesystem>
+#include <list>
```

Comment:
Since there are no other changes, I assume this can be removed.

Reply:
done, thanks!